modules/gdscript/gdscript_compiler.cpp

```diff
@@ -2452,6 +2452,25 @@ Error GDScriptCompiler::_parse_class_level(GDScript *p_script, const GDScriptPar
 				}
 #endif
 			} break;
+
+			case GDScriptParser::ClassNode::Member::GROUP: {
+				const GDScriptParser::AnnotationNode *annotation = member.annotation;
+				StringName name = annotation->export_info.name;
```

Comment:
[minor] Would it make sense to use a const reference here?
```cpp
const StringName& name = annotation->export_info.name;
```

Reply:
> I guess it doesn't matter much performance-wise?

Indeed, I've just double-checked. The copy-constructor of `StringName` just copies the internal reference to the string, but not the string contents itself: